portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,278 @@
+    background-size: 152% 100%;
+    background: #0409f8;
+    border: 2px solid #0409f8;
+}
+.btn:hover{
+    background: transparent;
+    border-color: #0409f8;
+    color: #0409f8;
+}
+.content{
+    position: relative;
+    top: 35%;
+    margin-top: 21%;
+}
+.content .title{
+    color: #1f1f25;
+    font-size: 15px;
+    text-transform: uppercase;
+    letter-spacing: 4px;
+    display: inline-block;
+    margin-bottom: 20px;
+    background: linear-gradient(120deg, #1c99fe 20.69%, #7644ff 50.19%,#fd4766 79.69%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+.content h1{
+    color: #1f1f25;
+    font-size: 75px;
+    font-weight: 900;
+    line-height: 90px;
+    text-transform: inherit;
+    width: 70%;
+}
+
 
-#greeting-container {
-  margin-top: 20px;
+.content h1 span{
+    color: #0409f8;
 }
+.content h1 span:hover{
+    color: #fff;
+    background-color: #0409f8;
+    border-radius: 25px;
+}
+.content h1 span:hover:after{
+    content:" Kafle";
+    
+}
+.content p{
+    width: 55%;
+    color: #757575;
+    margin-top: 25px;
+    margin-bottom: 30px;
+}
+.tag{
+    position: relative;
+}
+
+#self-image{
+    float: right;
+    margin-top: 100px;
+    width:300px;
+    height:400px;
+    border: #cfd3fd;
+}
+
+.skills{
+    margin-top: 8%;
+}
+
+
+.skills-grid{
+    display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 20px;
+  column-gap: 10px;
+  align-content: center;
+    margin: 4% 0 4% 0;
+}
+
+@media screen and (min-width: 600px) {
+    .skills-grid{
+    grid-template-columns: repeat(2, 1fr);
+    }
+}
+  
+@media screen and (min-width: 800px) {
+    .skills-grid{
+        grid-template-columns: repeat(3, 1fr);
+    }
+}
+
+
+
+.skills-grid > ul>li,h4{
+    list-style: none;
+    margin: 5px 15px 5px 35px;
+}
+.skills-grid>ul>img{
+    width: 80px;
+    margin-left: 40%;
+}
+.card{
+    background-color: #cfd3fd;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    width: 90%;
+    border-radius: 5px;
+    padding: 10px 10px 10px 10px;
+}
+
+.card:hover{
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.8);
+    cursor: pointer;
+}
+
+.projects-grid{
+    display: grid;
+    grid-template-columns: repeat(1, minmax(0, 1fr));
+    gap: 20px;
+    align-content: center;
+    margin: 4% 0 4% 0;
+}
+@media screen and (min-width: 800px) {
+    .projects-grid{
+        grid-template-columns: repeat(2, minmax(0, 1fr));
+        column-gap: 10px;
+        row-gap: 30px;
+    }
+}
+
+.contact{
+    background-color: #cfd3fd;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    width: 40%;
+    border-radius: 5px;
+    margin-top: 4%;
+}
+
+.contact>h1,form{
+    margin: 10% 10% 10% 10%;
+}
+
+.about-div{
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    margin-bottom: 4%;
+
+
+}
+
+@media screen and (min-width: 800px) {
+    .about-div{
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+.about-div >div> img{
```

Comment:
You only actually use the about-div once on the website, and it seems to be used more like an id than a class. If you're using a class, think more like a helper function. You'll probably use it in multiple places and in the case of a class, it's something that describes features and types of elements, rather than a specific element. If you plan to have multiple about-divs, that's a class. If you only have one, that's an id. 

You're also using a lot of > to find things. That's great for classes like you're using it! For an id, it might be better to use an id or class for those elements as well to make it clearer. Think of it from the readers point of view. It's much easier to search for a single id in the HTML than it is to find the class you're talking about and then find the div under it that has an img under it. Also, this will work for _any_ about-div > div > img, which includes any new ones you add in the future!

Reply:
I just learned about these.